beta/tests/test_sota_checkpoints.py

```diff
@@ -451,16 +482,77 @@ def test_eval(self, sota_checkpoints_dir, sota_data_dir, eval_test_struct: EvalR
         self.color_dict[eval_test_struct.model_name_], is_accuracy_within_thresholds = retval
         assert is_accuracy_within_thresholds
 
+    @pytest.mark.oveval
+    @pytest.mark.parametrize("eval_test_struct", ov_param_list, ids=ov_ids_list)
+    def test_openvino_eval(self, eval_test_struct: EvalRunParamsStruct, sota_checkpoints_dir, ov_data_dir, openvino):
+        if not openvino:
+            pytest.skip()
+        tf_checkpoint = PROJECT_ROOT / 'frozen_graph' / f'{eval_test_struct.model_name_}.pb'
+        ir_model_folder = PROJECT_ROOT / 'ir_models' / eval_test_struct.model_name_
+        config_folder = PROJECT_ROOT / 'tests' / 'tensorflow' / 'data' / 'ac_configs'
+        file = "main.py"
+        if eval_test_struct.sample_type_ == "segmentation":
+            file = "evaluation.py"
+        csv_result = f"{PROJECT_ROOT.parent}/{eval_test_struct.model_name_}.csv"
+        if eval_test_struct.model_name_.startswith("mask_"):
+            self.write_error_in_csv("AC does not support mask models yet", csv_result, eval_test_struct.model_name_)
+            pytest.fail("AC does not support mask models yet")
+        save_cmd = f"{sys.executable} examples/tensorflow/{eval_test_struct.sample_type_}/{file}" \
+                   f" --mode export --config {eval_test_struct.config_name_}" \
+                   f" --resume={sota_checkpoints_dir} " \
+                   f" --to-frozen-graph={tf_checkpoint}"
+        exit_code, err_str = self.run_cmd(save_cmd, cwd=PROJECT_ROOT)
+
+        if exit_code == 0:
+            mo_cmd = f"{sys.executable} mo.py" \
```

Comment:
Please use `'` instead of `"`. Check [styleguide](https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md)

Reply:
Done